packages/common-components/src/FileInput/FileInput.tsx

```diff
@@ -51,6 +51,9 @@ const useStyles = makeStyles(({ constants, palette, overrides }: ITheme) =>
     },
     error: {
       color: palette.error.main,
+      backgroundColor: palette.additional["gray"][3],
+      paddingLeft: 40,
+      paddingBottom: 4,
```

Comment:
To have the error vertically centered (my suggestion adds a 4px bottom and a 40px right that shouldn't matter)
```suggestion
      padding: "4px 40px",
```